dev/com.ibm.ws.cdi.third.party_fat/fat/src/com/ibm/ws/cdi12/fat/tests/HibernateSearchTest.java

```diff
@@ -10,65 +10,70 @@
  *******************************************************************************/
 package com.ibm.ws.cdi12.fat.tests;
 
-import static componenttest.annotation.SkipForRepeat.NO_MODIFICATION;
+import static componenttest.rules.repeater.EERepeatTests.EEVersion.EE8;
+import static org.junit.Assert.assertFalse;
 
 import java.io.File;
+import java.util.List;
+
+import org.junit.Assert;
 
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.asset.FileAsset;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.junit.AfterClass;
 import org.junit.Assert;
 import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
 import com.ibm.websphere.simplicity.ShrinkHelper;
-import com.ibm.ws.fat.util.LoggingTest;
 import com.ibm.ws.fat.util.SharedServer;
 import com.ibm.ws.fat.util.browser.WebBrowser;
+import com.ibm.ws.fat.util.browser.WebBrowserFactory;
 import com.ibm.ws.fat.util.browser.WebResponse;
 
-import componenttest.annotation.SkipForRepeat;
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.annotation.TestServlets;
 import componenttest.custom.junit.runner.FATRunner;
-import componenttest.custom.junit.runner.Mode;
-import componenttest.custom.junit.runner.Mode.TestMode;
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.impl.LibertyServerFactory;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.rules.repeater.EERepeatTests;
+import componenttest.topology.utils.FATServletClient;
 
-@Mode(TestMode.FULL)
-@SkipForRepeat(NO_MODIFICATION)
 @RunWith(FATRunner.class)
-public class HibernateSearchTest extends LoggingTest {
+public class HibernateSearchTest extends FATServletClient {
 
-    private static LibertyServer server;
+    public static final String HIBERNATE_SEARCH_APP_NAME = "hibernateSearchTest";
+    public static final String SERVER_NAME = "cdi20HibernateSearchServer";
 
-    @Override
-    protected SharedServer getSharedServer() {
-        return null;
-    }
+    @Server(SERVER_NAME)
+    @TestServlets({
+                    @TestServlet(servlet = cdi.hibernate.test.web.SimpleTestServlet.class, contextRoot = HIBERNATE_SEARCH_APP_NAME) 
+    })
+    public static LibertyServer server;
+
+    @ClassRule
+    public static RepeatTests r = EERepeatTests.with(SERVER_NAME, EE8);
 
     @BeforeClass
     public static void setUp() throws Exception {
 
-        WebArchive hibernateSearchTest = ShrinkWrap.create(WebArchive.class, "hibernateSearchTest.war")
+        //Hibernate Search Test
+        WebArchive hibernateSearchTest = ShrinkWrap.create(WebArchive.class, HIBERNATE_SEARCH_APP_NAME+".war")
                         .addPackages(true, "cdi.hibernate.test")
```

Comment:
If possible please could you switch from a hard coded package name to a real package from one of the classes? That way it still works if things get moved.